### PR TITLE
LavaMoat - UI upgrade - secure package naming

### PR DESCRIPTION
**Pull request by kumavis**
Upgrades :volcano:  `lavamoat-browserify` :volcano:  as used in the ui, to the latest version that uses secure package naming

### CODE REVIEW

File: development/build/scripts.js [hunk 1]
```diff
@@ -399,7 +399,7 @@ function createFactoredBuild({
       policyName: buildType,
       policyOverride: path.resolve(
         __dirname,
-        `../../lavamoat/browserify/${buildType}/policy-override.json`,
+        `../../lavamoat/browserify/policy-override.json`,
```
Comment (rekmarks) [inline development/build/scripts.js:402]: Why the move to a single policy override?

Comment (kumavis) [reply development/build/scripts.js:402]: less work maintaining a single override file

Comment (rekmarks) [reply development/build/scripts.js:402]: So I guess it doesn't matter if we override modules that exist for one build type but not others?

**Review by rekmarks — DISMISSED** [review]
LGTM.

File: development/build/scripts.js (hunk 1 above)
Comment (kumavis) [reply development/build/scripts.js:402]: correct. `lavapack` will even remove policy for packages that dont appear in the build.

**Review by brad-decker — DISMISSED** [review]
LGTM

**Review by brad-decker — APPROVED** [review]

**Comment by PeterYinusa** [comment]
LGTM

**Review by rekmarks — APPROVED** [review]

**Pull request merged**